src/charts/line.js

```diff
@@ -1510,6 +1565,21 @@ define(function(require){
             return this;
         };
 
+        /**
+         * Add custom horizontal lines to the Chart
```

Comment:
Please provide more descriptive comments for this one. Would be great if you explain what it does with an example:
https://github.com/eventbrite/britecharts/blob/master/src/charts/stacked-area.js#L1519

Reply:
Added more documentation. :)